javascript/javascript1/week2/script.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
+
+
+// Flight booking fullname function
+
+function getFullname(firstname, surname) {
+    return firstname + " " + surname;
+};
```

Comment:
Well written function! ⚡ You do not have to add a semicolon `;` at the end of functions, though. 😉